### DISCUSSION TITLE
Copilot completions do not work after most recent update PyCharm

### DISCUSSION THREAD

**Discussion by aaronwtr** [community/community#19255] [Copilot Conversations] [answered]
Whatever the context of my Python file, the Copilot window shows  `no completions were found` . Tried logging out and logging back in, restarting PyCharm, and restarting Copilot, all did not work. 

Any suggestions how to fix this?

**Comment by danhje** [comment]
I have the same issue. Have you tried checking all the boxes at https://github.com/settings/copilot? That seems to work for some people, though not for me.

**Reply by aaronwtr** [reply]
They are already checked. Thanks for the heads up though!

**Comment by aaronwtr** [comment, marked as answer]
It started working for me again after a recent Copilot update. I guess it was a bug related to the Copilot commercial roll out they did recently.

**Reply by tigran123** [reply]
No, it still does not work (as of 11 July 2024) even after all the updated to both copilot and PyCharm 2024.1.4 Pro

**Comment by andyzhanggr2020** [comment]
Thanks，I have the same issue.It‘s ok now in you way:)
